ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
In C/C++ code compiled to wasm, strings don't contain the "\0D", because stdout is in "text" mode, which ordinarily means that the C library translates newline into carriage-return-line-feed on Windows. However in wasm's case, the C library won't know that it's running on Windows, so it won't know that it has to do this. I think it's best if stdio.write handles this itself so that wasm programs don't have to worry about it.